package/tsconfig.json

```diff
@@ -1,5 +1,5 @@
 {
-  "extends": "astro/tsconfigs/strict",
+  "extends": "astro/tsconfigs/strictest",
```

Comment:
Does this change now mean all consumers have to use `strictest` too?

Reply:
Only integration authors should use this one